query-monitor.php

```diff
@@ -160,3 +160,4 @@ function change_dispatchers_shutdown_priority( array $dispatchers ) {
 require_once __DIR__ . '/qm-plugins/qm-alloptions/qm-alloptions.php';
 require_once __DIR__ . '/qm-plugins/qm-object-cache/qm-object-cache.php';
 require_once __DIR__ . '/qm-plugins/qm-apcu-cache/qm-apcu-cache.php';
+require_once __DIR__ . '/qm-plugins/qm-cron/qm-cron.php';
```

Comment:
Do we need to wrap this with `file_exists()` check to make sure that the very first deploy won't cause a spike of fatal errors?

Reply:
Wouldn't hurt!